### PR TITLE
added indication of backend working

### PR DESCRIPTION
**Pull request by faizjamil**
Related to #1799 

Add some text so devs know that the backend is working correctly since it is not evident.

### CODE REVIEW

File: app/readme.md [hunk 1]
```diff
@@ -60,6 +60,46 @@ docker-compose up --build
...
+If successful you will see something like this which indicates the backend is working
+```
+postgres_1  | 2021-08-11 16:26:48.928 UTC [36] LOG:  statement: COMMIT
+postgres_1  | 2021-08-11 16:26:49.931 UTC [38] LOG:  statement: SELECT 1
+postgres_1  | 2021-08-11 16:26:49.932 UTC [38] LOG:  statement: SET default_transaction_isolation TO DEFAULT
+postgres_1  | 2021-08-11 16:26:49.933 UTC [38] LOG:  statement: BEGIN ISOLATION LEVEL REPEATABLE READ
+postgres_1  | 2021-08-11 16:26:49.933 UTC [38] LOG:  statement: SELECT background_jobs.id, background_jobs.job_type, background_jobs.state, background_jobs.queued, background_jobs.next_attempt_after, background_jobs.try_count, background_jobs.max_tries, background_jobs.payload, background_jobs.failure_info
+postgres_1  |   FROM background_jobs
+postgres_1  |   WHERE background_jobs.next_attempt_after <= now() AND background_jobs.try_count < background_jobs.max_tries AND (background_jobs.state = 'pending' OR background_jobs.state = 'error') FOR UPDATE SKIP LOCKED
+postgres_1  | 2021-08-11 16:26:49.934 UTC [38] LOG:  statement: COMMIT
```
Comment (aapeliv) [inline app/readme.md:71]: Just one of these blocks is enough (from `SELECT 1` to `COMMIT`)

File: app/readme.md [hunk 2]
```diff
@@ -60,6 +60,46 @@ docker-compose up --build
 
 This will build the backend, database, and proxy, and start them up.
 
+If successful you will see something like this which indicates the backend is working
```
Comment (aapeliv) [inline app/readme.md]: "If successful you will see this message being repeated every few seconds which indicates that the backend is working"

**Review by aapeliv — CHANGES_REQUESTED** [review]

File: app/readme.md (hunk 2 above)
Comment (faizjamil) [reply app/readme.md]: Right of course, good call

**Comment by faizjamil** [comment]
May or may not have made an oopsie by leaving this branch behind `develop` by a few commits, apologies.

**Review by aapeliv — APPROVED** [review]

**Comment by aapeliv** [comment]
> May or may not have made an oopsie by leaving this branch behind `develop` by a few commits, apologies.

No worries. `develop` moves fast, it's easy to end up behind

**Pull request merged**